.changeset/strong-comics-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/cli-hydrogen': minor
+---
+
+Adds a "verbose" flag to the `shopify hydrogen build` command to outout a list of built files, among other logs from vite. These were previously displayed by default, but are now hidden when this flag is not present.
```

Comment:
Seems strange to see any references to @shopify/react-testing...

Reply:
@gfscott I'm not sure what might be causing this, but I think it is unrelated. Running `build` with `--path` is mostly broken with the demo-store (works with hello-world though).

see https://github.com/Shopify/cli/pull/346/files#diff-6bbf8e6805042dc2b7d82f6aa885b46c058abe94f1558528b97aec15bcfa78acR46

Reply:
OK, good to know — this is extremely likely to be operator error on my part anyway 😅
Concept looks good! 👍🏻

Reply:
> @rennyG we'll need to add this flag to the docs for the build command.

Thanks @cartogram! Due to the cuts on our team, we're moving into a new flow for determining capacity and handling issues. It involves triaging and prioritizing in sprints. I'll still handle Hydrogen tickets in some capacity, though. 

Can you [create an issue for the docs update using this template](https://github.com/Shopify/shopify-dev/issues/new?assignees=&labels=discipline-content&template=new_doc.md)? Give a deadline, as our team requires that in order to triage the issue. Otherwise, it could languish in backlogland